web/src/redcross/discussionModule/components/DiscussionComponent.ts

```diff
@@ -18,8 +18,8 @@ export class DiscussionComponent {
     private posts: Post[];
     private currentUser: Author;
 
-    constructor(public dds: DiscussionDataService,
-                public md: MatDialog
+    constructor(private dds: DiscussionDataService,
```

Comment:
Rename this component to DiscussionContainerComponent. Also rename the html and less

Also move the AddPostDialog class into a separate file